tuftool/src/create.rs

```diff
@@ -89,21 +91,17 @@ impl CreateArgs {
                 .context(error::InitializeThreadPool)?;
         }
 
-        let root_buf = std::fs::read(&self.root).context(error::FileRead { path: &self.root })?;
-        let root = serde_json::from_slice::<Signed<Root>>(&root_buf)
-            .context(error::FileParseJson { path: &self.root })?
-            .signed;
-        let mut root_sha256 = [0; SHA256_OUTPUT_LEN];
-        root_sha256.copy_from_slice(digest(&SHA256, &root_buf).as_ref());
-        let root_length = root_buf.len() as u64;
+        let root_digest = RootDigest::digest_root_file(&self.root)?;
+        let key_pairs = keys_for_root(&self.keys, &root_digest.root)?;
+        ensure!(!key_pairs.is_empty(), error::KeysNotFoundInRoot {});
```

Comment:
This can maybe get rolled into `keys_for_root`.

Reply:
Looks like this was missed?

This error is thrown in `create` but not `sign` or `refresh` so it'd be nicet o move it into `fn load_keys`.

Reply:
got it this time.